### ISSUE TITLE
Takes 1 hour to do 10K iterations !

### ISSUE THREAD

**Issue by anaseinea** [Easymode-ai/ComfyUI-BPT#3]
I am using windows portable version of comfy, when I run I see these logs:

`Converting Trimesh to PyMeshLab...

Exporting to temporary file...

Loading into PyMeshLab...

reducer...

D:\AI\COMFYU~1\python_embeded\Lib\site-packages\x_transformers\x_transformers.py:435: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled = False)
D:\AI\COMFYU~1\python_embeded\Lib\site-packages\x_transformers\x_transformers.py:461: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled = False)
[2025-03-16 12:25:27,587] [INFO] [real_accelerator.py:222:get_accelerator] Setting ds_accelerator to cuda (auto detect)
FETCH ComfyRegistry Data: 10/78
test.c
LINK : fatal error LNK1181: cannot open input file 'aio.lib'
test.c
LINK : fatal error LNK1181: cannot open input file 'cufile.lib'
"C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" /TcC:\Users\anasa\AppData\Local\Temp\tmpycwvi9th\test.c /FoC:\Users\anasa\AppData\Local\Temp\tmpycwvi9th\test.obj
--- Logging error ---
"C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\bin\HostX86\x64\link.exe" /nologo /INCREMENTAL:NO /LTCG /MANIFEST:EMBED,ID=1 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.22621.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x64" aio.lib C:\Users\anasa\AppData\Local\Temp\tmpycwvi9th\test.obj /OUT:C:\Users\anasa\AppData\Local\Temp\tmpycwvi9th\a.out.exe
--- Logging error ---
"C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" /TcC:\Users\anasa\AppData\Local\Temp\tmphng1uzdi\test.c /FoC:\Users\anasa\AppData\Local\Temp\tmphng1uzdi\test.obj
--- Logging error ---
"C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\bin\HostX86\x64\link.exe" /nologo /INCREMENTAL:NO /LTCG /MANIFEST:EMBED,ID=1 "/LIBPATH:C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.4" "/LIBPATH:C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.4\lib64" "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.42.34433\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.22621.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x64" cufile.lib C:\Users\anasa\AppData\Local\Temp\tmphng1uzdi\test.obj /OUT:C:\Users\anasa\AppData\Local\Temp\tmphng1uzdi\a.out.exe
--- Logging error ---
W0316 12:25:31.393000 25252 D:\AI\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\distributed\elastic\multiprocessing\redirects.py:29] NOTE: Redirects are currently not supported in Windows or MacOs.
Point cloud encoder: miche-256-feature | freeze: False
{'model': {'target': 'bpt.miche.michelangelo.models.tsal.asl_pl_module.AlignedShapeAsLatentPLModule', 'params': {'shape_module_cfg': {'target': 'bpt.miche.michelangelo.models.tsal.sal_perceiver.AlignedShapeLatentPerceiver', 'params': {'num_latents': 256, 'embed_dim': 64, 'point_feats': 3, 'num_freqs': 8, 'include_pi': False, 'heads': 12, 'width': 768, 'num_encoder_layers': 8, 'num_decoder_layers': 16, 'use_ln_post': True, 'init_scale': 0.25, 'qkv_bias': False, 'use_checkpoint': True}}, 'aligned_module_cfg': {'target': 'bpt.miche.michelangelo.models.tsal.clip_asl_module.CLIPAlignedShapeAsLatentModule', 'params': {'clip_model_version': './checkpoints/clip/clip-vit-large-patch14'}}, 'loss_cfg': {'target': 'bpt.miche.michelangelo.models.tsal.loss.ContrastKLNearFar', 'params': {'contrast_weight': 0.1, 'near_weight': 0.1, 'kl_weight': 0.001}}, 'optimizer_cfg': {'optimizer': {'target': 'torch.optim.AdamW', 'params': {'betas': [0.9, 0.99], 'eps': 1e-06, 'weight_decay': 0.01}}, 'scheduler': {'target': 'bpt.miche.michelangelo.utils.trainings.lr_scheduler.LambdaWarmUpCosineFactorScheduler', 'params': {'warm_up_steps': 5000, 'f_start': 1e-06, 'f_min': 0.001, 'f_max': 1.0}}}}}}
 custom path :
D:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BPT

 custom path :
D:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BPT

FETCH ComfyRegistry Data: 15/78
 custom path :
D:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BPT

 custom path :
D:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BPT

BPT Model Path: D:\AI\ComfyUI_windows_portable\ComfyUI\models\bpt\bpt-8-16-500m.pt

D:\AI\COMFYU~1\python_embeded\Lib\site-packages\pytorch_custom_utils\save_load.py:53: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  pkg = torch.load(str(path), map_location = 'cpu')
Process: 0:   0%|                                                                            | 0/10000 [00:00<?, ?it/s]contextlib.py:105: FutureWarning: `torch.backends.cuda.sdp_kernel()` is deprecated. In the future, this context manager will be removed. Please see `torch.nn.attention.sdpa_kernel()` for the new context manager, with updated signature.
  self.gen = func(*args, **kwds)
FETCH ComfyRegistry Data: 20/78
Process: 0:   0%|                                                                  | 9/10000 [00:04<1:09:17,  2.40it/s]`

do I need to worry about the:
test.c
LINK : fatal error LNK1181: cannot open input file 'aio.lib'
test.c
LINK : fatal error LNK1181: cannot open input file 'cufile.lib'
?

the other thing is as you see the speed is 2.4 iterations a second and 10K iterations regardless of the inputs of the node.
I am using RTX 4070 laptop, 8GB VRAM, 64GB RAM.
